From 65511f92122507b8bc478365d91f75ae21ce58f9 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Tue, 10 Mar 2020 15:12:32 +0700
Subject: [PATCH 191/351] arm64: defconfig: Add CONFIG_TMPFS_POSIX_ACL

This patch enables config CONFIG_TMPFS_POSIX_ACL in defconfig to
support addtional access rights for users and groups for ACLs
specifically for tmpfs filesystem.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 4c1b23d..d60c97d 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -375,3 +375,4 @@ CONFIG_SATA_AHCI=y
 CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_INPUT_UINPUT=m
 CONFIG_INPUT_MOUSEDEV=m
+CONFIG_TMPFS_POSIX_ACL=y
-- 
2.7.4

